src/client/client4.js

```diff
@@ -524,8 +525,15 @@ export default class Client4 {
     getProfilesInChannel = async (channelId, page = 0, perPage = PER_PAGE_DEFAULT) => {
         this.trackEvent('api', 'api_profiles_get_in_channel', {channel_id: channelId});
 
+        const serverVersion = Client4.getServerVersion();
+        let queryStringObj;
+        if (isMinimumServerVersion(serverVersion, 4, 7)) {
+            queryStringObj = {in_channel: channelId, page, per_page: perPage, sort: 'status'};
+        } else {
```

Comment:
You could create this object once and add the status property if needed

Reply:
I could, but I my thought was that, since this code should be deleted in the future, this would make it easier for whomever deletes the check in the future to immediately see it can simply be deleted.

Reply:
They really have to just switch from `let` => `const` and delete a code block.